src/site/stages/build/options.js

```diff
@@ -22,6 +22,7 @@ const COMMAND_LINE_OPTIONS_DEFINITIONS = [
   { name: 'destination', type: String, defaultValue: null },
   { name: 'content-deployment', type: Boolean, defaultValue: false },
   { name: 'content-directory', type: String, defaultValue: defaultContentDir },
+  { name: 'local-proxy-rewrite', type: Boolean, defaultValue: false },
```

Comment:
The local proxy server only starts when this argument is passed, because I didn't want us to hit the production site (which is necessary for TeamSite dev) unless we explicitly tell it that's what we want it to do.